cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/plugins/test/TestPlugin.java

```diff
@@ -35,18 +35,43 @@
 
   @Override
   public String call() throws Exception {
-    if (config.timeout % 2 == 0) {
-      return config.name;
+    if (config.timeout != null) {
+      if (config.timeout % 2 == 0) {
```

Comment:
this is strangely named, its not really a timeout at all but a flag to determine what gets returned. Can we rename it to something more descriptive?